znai-docs/znai/release-notes/1.50/add-2021-06-15-python-function.md

```diff
@@ -1 +1 @@
-* Add [include-python](snippets/python#function-content) to include function's content
\ No newline at end of file
+* Add [include-python](snippets/python#content) to include function's content
```

Comment:
Reminds me. Could you please add a new file like this pointing to a new feature you added?

Reply:
Done.